docker/nats/build-push.sh

```diff
@@ -6,7 +6,7 @@ set -e
 # --- Config ---
 REPOSITORY="wundergraph/cosmo"
 IMAGE_NAME="nats"
-IMAGE_TAG="2.11.0"
+IMAGE_TAG="2.11.0-alpine"
```

Comment:
What was the reason to change this?

Reply:
normal image does not have a linux distro, and thus cannot install `curl`, for the healthcheck

Reply:
as for the internal tag, i thought it best to keep them aligned? i can make it just `2.11.0`

Reply:
Thanks for clarification